src/trigger/helpers/transcript-extractors/__tests__/fixtures/limitless.html

```
@@ -0,0 +1,22 @@
+<html>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)